tasks/main.yml

```diff
@@ -27,11 +27,7 @@
 
 - name: system packages | install selinux utilities
   become: yes
-  yum:
-    name: "{{ item }}"
+  package:
+    name: ["libselinux-python", "libsemanage-python", "policycoreutils-python"]
```

Comment:
Could you expand this list to one item per-line. Also `"` quotes aren't needed

Reply:
Hi Simon,

Thank you for the review,
Please check if the latest changes are matching what you had in mind.